### ISSUE TITLE
HoverCard and Popover are not working with Switch

### ISSUE THREAD

**Issue by dan-iliescu** [mantinedev/mantine#5072]
### What package has an issue

@mantine/core

### Describe the bug

Wrapping the Switch inside the provided demo doesn't work at all, meaning I get no text on hover, but changing from Switch to Button makes it work.

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by dan-iliescu** [comment]
Only wrapping the Switch inside a div works for HoverCard, but not for Popover.

**Comment by rtivital** [comment]
The fix will be available with 7.1.4 release:

```tsx
export function WithSwitch() {
  return (
    <div style={{ padding: 40 }}>
      <HoverCard width={280} shadow="md">
        <HoverCard.Target refProp="rootRef" eventPropsWrapperName="wrapperProps">
          <Switch label="Switch label" />
        </HoverCard.Target>
        <HoverCard.Dropdown>
          <p>
            Hover card is revealed when user hovers over target element, it will be hidden once
            mouse is not over both target and dropdown elements
          </p>
        </HoverCard.Dropdown>
      </HoverCard>
    </div>
  );
}
```
